wochenende_postprocess.sh

```diff
@@ -82,17 +87,24 @@ echo "INFO: Completed Sambamba depth and filtering"
 
 
 # Plots
-echo "INFO: Started Wochenende plot"
-cd $bamDir
-cd plots
-cp ../*_window.txt . 
-cp ../*_window.txt.filt.csv .
-bash runbatch_wochenende_plot.sh >/dev/null 2>&1
-#wait
-echo "INFO: Sleeping for " $sleeptimer
-sleep $sleeptimer
-cd $bamDir
-echo "INFO: Completed Wochenende plot"
+if [ $1 = "--no-plot" ] 
+then
+    echo "INFO: Found --no-plot argument: Plot mode disabled"
```

Comment:
Why do we state this twice?

Reply:
cheers for looking Tobias.
If you're just referring to the code snippet you have, if I'm correct, line 90 does the if statement but produces no on screen output, while 92 prints the logging statement INFO for the user.
For me, that's not a problem. Or do you mean another section ?

Reply:
It seems to be a duplicate of line 35